helix-view/src/editor.rs

```diff
@@ -276,7 +276,13 @@ impl Default for StatusLineConfig {
         Self {
             left: vec![E::Mode, E::Spinner, E::FileName],
             center: vec![],
-            right: vec![E::Diagnostics, E::Selections, E::Position, E::FileEncoding],
+            right: vec![
+                E::Diagnostics,
+                E::Selections,
+                E::TotalCharsSelected,
+                E::Position,
+                E::FileEncoding,
+            ],
```

Comment:
I think this should be disabled by default

Reply:
does this vector contain the statusline elements that are active by default?
if so, do i need to also create a means to toggle this element?

Reply:
You can undo this particular change. The statusline is configurable: https://docs.helix-editor.com/configuration.html#editorstatusline-section